themes/marvelous-snuffle/static/css/style.css

```diff
@@ -1,39 +1,51 @@
 /* ============================================================= COLORS ** ============================================================= */
+:root{
+    --global-font-color:  #e0e0e0;
+    --global-button-background-color: #283747;
+    --background-color-2: #212121;
+    --global-border-color: var(--global-font-color);
+    --global-border: 1px solid var(--global-border-color);
+    --global-font-size: 13px;
+    --global-padding: 15px;
```

Comment:
Please don't use px as a unit, I would recommend em, or rem. Which will be more responsive.

Reply:
Ok